Source/SSTSmallStrings.m

```diff
@@ -50,13 +55,27 @@ id SSTJsonForName(NSString *name)
     return keyToString; // Avoid -copy to be a bit faster
 }
 
-NSString *SSTStringForKey(NSString *key)
-{
+NSString *SSTStringForKeyWithBundleAndSubdirectoryAndTargetName(NSString *key, NSBundle *bundle, NSString *subdirectory, NSString *targetName) {
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
-        sKeyToString = SSTCreateKeyToString();
+        sKeyToString = SSTCreateKeyToString(bundle, subdirectory, targetName);
     });
     // Haven't tested with CFBundleAllowMixedLocalizations set to YES, although it seems like that'd be handled by the
     // NSLocalizedString fallback
     return sKeyToString[key] ?: NSLocalizedString(key, @"");
 }
+
+NSString *SSTStringForKeyWithBundleAndSubdirectory(NSString *key, NSBundle *bundle, NSString *subdirectory)
```

Comment:
We intentionally avoid having the NSBundle as part of the public function because SmallStrings should always be used with just one localization file per language, not spread out into multiple bundles. That's because you don't really get compression benefits if the data is split up into multiple files before being compressed. Also, because of the minimum files size: https://docs.emergetools.com/docs/avoid-many-files you can actually end up increasing app size by splitting the translation keys out into a separate file (like SmallStrings does) if you are spreading localizations across many files.

Reply:
Will refactor with this approach in mind